NewForce-at-Mountwest/bangazon-site-the-grave-trolls
# Initial Setup and .gitignore

# Description

Performed initial setup for project, adjusted PaymentType model, and adjusted the .gitignore

# Testing Instructions

- Pull down changes to a new branch
- Open code in Visual Studio
- Open Package Manager Console
- Run `add-migration CreateDatabase` to add a new migration to our project
- Run `update-database` to perform this migration's changes and create our initial database, as well as seed data
- Open the SQL Server Object Explorer and ensure that the database and its tables were properly created
